lib/my_john_deere_api/request/create/base.rb

```diff
@@ -63,5 +63,16 @@ def headers
         'Content-Type'  => 'application/vnd.deere.axiom.v3+json'
       }
     end
+
+    ##
+    # Return response's Location header, and fail loudly if unavailable
+
+    def location_header
+      if response['location']
+        response['location'].split('/platform').last
+      else
+        raise MissingLocationHeaderError.new(response)
+      end
+    end
```

Comment:
Are you concerned that the location header might be blank, or just that it might be missing altogether? Using `fetch` would raise an error on a missing key.
```suggestion
    def location_header
      response.fetch('location').split('/platform').last
    rescue KeyError
      raise MissingLocationHeaderError.new(response)
    end
```

Reply:
I guess I'm not opposed to making the change, but I'm not sure what clarity it adds to pass through an extra error. If anything, I have to remember that a failed fetch throws a KeyError, whereas my original code makes the cause/effect more obvious (to my mind)

Reply:
Also, I didn't know we could do suggested changes, I should have been using this :)

Reply:
This was a "soft" suggestion. It's a question of style rather than correctness, and the alternate version clearly didn't appeal to you. :)